### PR TITLE
OKTA-599935 Update logo requirements in OIN Manager

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Update OIN logo requirements as per fixes in OIN Manager.
- **Is this PR related to a Monolith release?** No

### Preview:
https://6439b05a999d573b619a8b38--reverent-murdock-829d24.netlify.app/docs/guides/submit-app-prereq/main/#logo-guidelines

### Resolves:

* [OKTA-599935](https://oktainc.atlassian.net/browse/OKTA-599935)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md [hunk 1]
```diff
@@ -38,26 +38,22 @@ Before you submit your integration, ensure that your integration doesn't use Okt
...
 ### Logo tips
 
-* If your logo isn't in one of the acceptable formats, ask your design team to convert your existing logo to the preferred PNG format. If you have difficulty with converting your image to an acceptable format, contact the [OIN team](mailto:oin@okta.com).
+* If your logo isn't in one of the acceptable formats, ask your design team to convert your existing logo to the preferred PNG format.
 
-* If your logo isn't square, consider using your website favicon. Alternatively, you can use the first letter of your app wordmark and convert it to a square image.
+* A square logo is preferred. If your logo isn't square, consider using your website favicon. Alternatively, you can use the first letter of your app wordmark and convert it to a square image.
 
-* You can use a slightly non-square icon, but consider the scaling that occurs when your logo resizes to fit a 200 x 200 pixel area.
+* If you submit an image that is outside of the four allowed dimensions, your logo might be distorted when it's displayed in the catalog. OIN app icons are placed in a 200 x 200 pixel grid, so consider the scaling that occurs when your logo resizes to fit the grid.
```
Comment (tomaszwrobel-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md:56]: Technically they won't be able to submit anything outside of those four dimensions as the form will reject other dimensions

**Review by tomaszwrobel-okta — APPROVED** [review]
LGTM!

File: packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md [hunk 2]
```diff
@@ -38,26 +38,26 @@ Before you submit your integration, ensure that your integration doesn't use Okt
...
     * A colored background is acceptable if it's a part of the logo color scheme
+* Submit a logo image with sharp corners (no rounded corners).
 
 ### Logo tips
 
-* If your logo isn't in one of the acceptable formats, ask your design team to convert your existing logo to the preferred PNG format. If you have difficulty with converting your image to an acceptable format, contact the [OIN team](mailto:oin@okta.com).
+* If your logo isn't in one of the acceptable formats, ask your design team to convert your existing logo to the preferred PNG format.
 
-* If your logo isn't square, consider using your website favicon. Alternatively, you can use the first letter of your app wordmark and convert it to a square image.
+* If your image isn't in one of the four allowed dimensions, ask your design team to scale your logo appropriately. The OIN Manager doesn't allow you to submit any other dimensions.
```
Comment (isaacdurham-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md:58]: Consider dropping the number just to reduce friction if they change the allowed dimensions. 
```suggestion
* If your image isn't in one of the allowed dimensions, ask your design team to scale your logo appropriately. The OIN Manager doesn't allow you to submit any other dimensions.
```

**Review by isaacdurham-okta — APPROVED** [review]

**Review by ykrestyanov-okta — APPROVED** [review]

**Pull request merged**